doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/index.md

```diff
@@ -5,22 +5,84 @@ permalink: /:path/
 sort_idx: 10
 ---
 
-- [Intruduction](#introduction)
-- [Build configuration](#build-configuration)
+* TOC
+{:toc}
 
-# Introduction
+*C SDK* is a portable, lightweight and fast client-side tool that provides the full set of Kaa
```

Comment:
s/client-side tool that provides the full set of Kaa application-level features/library that provides APIs for Kaa clients to conveniently use various platform features/